spring-cloud-gcp-autoconfigure/src/main/java/com/google/cloud/spring/autoconfigure/storage/GcpStorageProperties.java

```diff
@@ -38,11 +38,36 @@ public Credentials getCredentials() {
 
   private String projectId;
 
+  /**
+   * Universe domain of the client which is part of the host that is formatted as
+   * `https://${service}.${universeDomain}`.
+   */
+  private String universeDomain;
+
+  /** Host of the Storage client that is formatted as `https://${service}.${universeDomain}/`. */
```

Comment:
I see that we have an ending slash here but not in the format example above for universeDomain, which one we should go for?

Reply:
Good catch, modified the documentation to be consistent.